lib/modem_info/modem_info.c

```diff
@@ -20,168 +21,257 @@ LOG_MODULE_REGISTER(modem_info);
 
 #define INVALID_DESCRIPTOR	-1
 
-#define AT_CMD_CESQ		"AT%CESQ"
+#define AT_CMD_CESQ		"AT+CESQ"
 #define AT_CMD_CESQ_ON		"AT%CESQ=1"
 #define AT_CMD_CESQ_OFF		"AT%CESQ=0"
 #define AT_CMD_CESQ_RESP	"%CESQ"
 #define AT_CMD_CURRENT_BAND	"AT%XCBAND"
+#define AT_CMD_SUPPORTED_BAND	"AT%XCBAND=?"
 #define AT_CMD_CURRENT_MODE	"AT+CEMODE?"
 #define AT_CMD_CURRENT_OP	"AT+COPS?"
-#define AT_CMD_CELLID		"AT+CEREG?"
+#define AT_CMD_NETWORK_STATUS	"AT+CEREG?"
 #define AT_CMD_PDP_CONTEXT	"AT+CGDCONT?"
 #define AT_CMD_UICC_STATE	"AT%XSIM?"
 #define AT_CMD_VBAT		"AT%XVBAT"
 #define AT_CMD_TEMP		"AT%XTEMP"
 #define AT_CMD_FW_VERSION	"AT+CGMR"
 #define AT_CMD_CRSM		"AT+CRSM"
 #define AT_CMD_ICCID		"AT+CRSM=176,12258,0,0,10"
+#define AT_CMD_SYSTEMMODE	"AT%XSYSTEMMODE?"
 #define AT_CMD_SUCCESS_SIZE	5
-#define RSRP_PARAM_INDEX 0
-#define RSRP_PARAM_COUNT 2
-#define RSRP_OFFSET_VAL 141
 
-#define BAND_PARAM_INDEX 0 /* Index of desired parameter in returned string */
-#define BAND_PARAM_COUNT 1 /* Number of parameters returned from modem */
+#define RSRP_DATA_NAME		"rsrp"
+#define CUR_BAND_DATA_NAME	"curBand"
+#define SUP_BAND_DATA_NAME	"supBands"
+#define UE_MODE_DATA_NAME	"ueMode"
+#define OPERATOR_DATA_NAME	"mccmnc"
+#define MCC_DATA_NAME		"mcc"
+#define MNC_DATA_NAME		"mnc"
+#define AREA_CODE_DATA_NAME	"areaCode"
+#define CELLID_DATA_NAME	"cellID"
+#define IP_ADDRESS_DATA_NAME	"ip"
+#define UICC_DATA_NAME		"uicc"
+#define BATTERY_DATA_NAME	"btryV"
+#define TEMPERATURE_DATA_NAME	"temp"
+#define MODEM_FW_DATA_NAME	"mFW"
+#define ICCID_DATA_NAME		"iccid"
```

Comment:
Is it github, or there is an alignment issue?

Reply:
Github I believe. Not on my end at least.